pom.xml

```diff
@@ -32,6 +42,27 @@
 			<artifactId>spring-boot-starter-test</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-validation</artifactId>
+			<version>RELEASE</version>
```

Comment:
To jeszcze działa?

Reply:
Ups, miało być bez wersji, powinna być z parenta. `LATEST` i `RELEASE` chyba nie działają dla wersji pluginów ale w dependency już tak.